server-spi-private/src/main/java/org/keycloak/validate/BuiltinValidators.java

```diff
@@ -19,17 +19,7 @@
 
 package org.keycloak.validate;
 
-import org.keycloak.validate.validators.DoubleValidator;
-import org.keycloak.validate.validators.EmailValidator;
-import org.keycloak.validate.validators.IntegerValidator;
-import org.keycloak.validate.validators.LengthValidator;
-import org.keycloak.validate.validators.LocalDateValidator;
-import org.keycloak.validate.validators.NotBlankValidator;
-import org.keycloak.validate.validators.NotEmptyValidator;
-import org.keycloak.validate.validators.OptionsValidator;
-import org.keycloak.validate.validators.PatternValidator;
-import org.keycloak.validate.validators.UriValidator;
-import org.keycloak.validate.validators.ValidatorConfigValidator;
+import org.keycloak.validate.validators.*;
```

Comment:
Can you please revert this change?

Reply:
Sure, that was just the auto-optimize.

Just curious, are there specific benefits with the explicit imports?

Reply:
Not really it is more about making more explicit the dependencies you have. We avoid using wildcards even though we don't have any automation to enforce a coding style.